### PR TITLE
@daimo/userop wrap TODOs

### PR DESCRIPTION
**Pull request by nalinbhardwaj**
- move expo-enclave/example to be its own app
- Finish todos from #25
- Potentially discover bug in `sendUserOperation`??? https://goerli.basescan.org/tx/0x576376aafa9208434f51c0906b732ef9f56e6db087b6293bd7d0347177b2cb41

### CODE REVIEW

File: apps/daimo-testbed/eas.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    "version": ">= 3.9.2"
+  },
+  "build": {
+    "sim": {
+      "ios": {
+        "buildConfiguration": "Debug",
+        "resourceClass": "m-medium",
+        "simulator": true
+      }
+    },
```
Comment (dcposch) [inline apps/daimo-testbed/eas.json:12]: 👍

File: apps/daimo-testbed/package.json [hunk 2]
```diff
@@ -9,25 +9,22 @@
     "web": "expo start --web"
   },
   "dependencies": {
-    "@daimo/contract": "*",
+    "@daimo/expo-enclave": "*",
     "@daimo/userop": "*",
-    "eas-cli": "^3.12.0",
-    "expo": "~48.0.15",
+    "@daimo/contract": "*",
```
Comment (dcposch) [inline apps/daimo-testbed/package.json:14]: 👍 👍

File: packages/daimo-expo-enclave/package.json [hunk 3]
```diff
@@ -25,7 +26,10 @@
   "homepage": "https://github.com/daimo-eth/daimo#readme",
   "repository": "https://github.com/daimo-eth/daimo/tree/master/packages/daimo-expo-enclave",
   "author": "Daimo",
-  "dependencies": {},
+  "dependencies": {
+    "@daimo/contract": "*",
+    "@daimo/userop": "*"
```
Comment (dcposch) [inline packages/daimo-expo-enclave/package.json:31]: my bad, missed this in the big refactor

File: packages/daimo-userop/src/index.ts [hunk 4]
```diff
@@ -52,7 +60,21 @@ export class DaimoAccount {
...
     );
 
-    return new DaimoAccount(dryRun, client, daimoBuilder);
+    const erc20 = getContract({
+      abi: Contracts.erc20ABI,
+      address: tokenAddress,
+      publicClient,
+    });
+
+    const tokenDecimals = await erc20.read.decimals();
```
Comment (dcposch) [inline packages/daimo-userop/src/index.ts:69]: 👍 👍

**Review by dcposch — APPROVED** [review]
LGreatTM

**Comment by nalinbhardwaj** [comment]
thanks for the rebase!

**Pull request merged**